tests/benchmarks/thread_metric/prj.conf

```diff
@@ -23,3 +23,6 @@ CONFIG_PICOLIBC_USE_MODULE=y
 
 # Disable Thread Local Storage for better context switching times
```

Comment:
Unrelated to this PR obviously but this makes it sound as if we're tweaking options for the *only* purpose of getting better numbers in the benchmark. I think the wording of this one should be more along the lines of: disabling TLS as it's not relevant/needed for the benchmark